### PR TITLE
Roni reward ads

### PR DESCRIPTION
**Pull request by ronigrr**

### CODE REVIEW

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt [hunk 1]
```diff
@@ -410,7 +420,7 @@ class FragmentArcadeModeScreen : Fragment(), View.OnClickListener, IFinishTimer
             }
             if (isDivideZero || isFraction) {
                 AudioManager.startBuzzerSound()
-                showFinishDialog(Constants.LOSE_DIALOG)
+                //showFinishDialog(Constants.LOSE_DIALOG)
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt]: why to comment??

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt [hunk 2]
```diff
@@ -445,7 +455,7 @@ class FragmentArcadeModeScreen : Fragment(), View.OnClickListener, IFinishTimer
                 } else {
                     //you loose
                     Handler().postDelayed({
-                        showFinishDialog(Constants.LOSE_DIALOG)
+                        //showFinishDialog(Constants.LOSE_DIALOG)
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt:457]: also?

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt [hunk 3]
```diff
@@ -12,31 +12,34 @@ import com.example.makeiteven2.R
...
             mCallBack.onLevelsFragmentBackPressed()
         }
+        initLevelsAdapter()
         initLevels()
         initRecyclerView()
 
         return rootView
     }
 
+    private fun initLevelsAdapter() {
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt]: ????? empty fun?

File: app/src/main/java/com/example/makeiteven2/intefaces/IEndDialogBtnClicked.kt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.example.makeiteven2.intefaces
+
+import android.view.View
+
+interface IEndDialogBtnClicked {
```
Comment (ilya3a) [inline app/src/main/java/com/example/makeiteven2/intefaces/IEndDialogBtnClicked.kt]: change the name to Listener in the end - more readable

**Review by ilya3a — DISMISSED** [review]

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt (hunk 3 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt]: oo come on I just forgot to remove!

File: app/src/main/java/com/example/makeiteven2/intefaces/IEndDialogBtnClicked.kt (hunk 4 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/intefaces/IEndDialogBtnClicked.kt]: done

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt (hunk 2 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt:457]: I'm not sure if to show dialog here or not, see after merge it's in your fragments responsibility

File: app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt (hunk 1 above)
Comment (ronigrr) [reply app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt]: removed

**Pull request merged**